ui/raidboss/data/02-arr/raid/t1.js

```diff
@@ -119,4 +119,71 @@
       },
     },
   ],
+  timelineReplace: [
+    {
+      'locale': 'de',
+      'replaceSync': {
+        'Ads': 'Abwehrsystem',
+        'Caduceus': 'Caduceus',
+        'Dark Matter Slime': 'Dunkelmaterien-Schleim',
+        'The Allagan megastructure': 'Allagische Superstruktur',
+      },
+      'replaceText': {
+      },
+      '~effectNames': {
+      },
+    },
+    {
+      'locale': 'fr',
+      'replaceSync': {
+        'Ads': 'Sphère de contrôle',
+        'Caduceus': 'Caducée',
+        'Dark Matter Slime': 'Gluant de matière sombre',
+        'The Allagan megastructure': 'Mégastructure allagoise',
+      },
+      'replaceText': {
+      },
+      '~effectNames': {
+      },
```

Comment:
Poison violent
it's debuff Regorge when you are in Spit on Caduceus

Reply:
Same as for Giftpollen, its not used in the js/txt files so it does not need to be included,